src/resources/resource.js

```diff
@@ -0,0 +1,43 @@
+export default function resource(path) {
+  return client => (
+    {
+      path,
+      client,
+
+      findAll({ filter = null,
+                orderBy = null, top = 400, skip = 0 } = {}) {
+        client.logger.info('findAll', { path: this.path, filter, orderBy, top, skip });
+        return client.get(this.path, { $filter: filter, $orderby: orderBy, $top: top, $skip: skip })
```

Comment:
yeah j, it is different, docs says

> {URI}/stockitem/search?q={query}

but does not describe query object format